calendar/page.js

```diff
@@ -218,6 +218,7 @@ class CalendarHandler {
     TZDate = this.calendar.getDate().constructor;
 
     this.calendar.on('clickEvent', async (info) => {
+      focusWidget();
```

Comment:
Why this is needed?

Reply:
Without this, the calendar widget doesn't get focus if you click or double-click an event. This breaks the new `viewAsCard` command because it opens the selected record in the active section, and if the calendar isn't the active section, the wrong record is shown in the popup.